components/support/webextensions/build.gradle

```diff
@@ -37,7 +37,6 @@ dependencies {
     implementation project(':browser-state')
     implementation project(':support-base')
     implementation project(':support-ktx')
-    implementation Dependencies.androidx_lifecycle_extensions
```

Comment:
I think this was added due to `WebExtensionPopupFeature`. Can we verify that works? (Sample browser -> install ublock -> click on ublock browser action)

Reply:
Yes, it still works.